common/components/register_model/spec.yaml

```diff
@@ -55,9 +56,14 @@ inputs:
   model_info_path:
     type: uri_file
     optional: true
-    description: A JSON file to provide model registration specific settings (name, type, description, tags, properties). This will take priority over values provided through component inputs
+    description: A JSON file to provide model registration specific settings (name, type, description, tags, properties).
```

Comment:
can we name it metadata then?

Reply:
@vizhur please let us know if this looks good. resolving